web_monitoring/db.py

```diff
@@ -760,3 +785,17 @@ def get_version_by_versionista_id(self, versionista_id):
         # Make result look like the result of `get_version` rather than the
         # result of `list_versions`.
         return {'data': result['data'][0]}
+
+    def validate_credentials(self):
+        """
+        Validate that the DB Client is authorized for the provided host
+
+        Parameters
+        ----------
+        none
+
+        Returns
+        -------
+        none
```

Comment:
Same here — no need to include these sections if there’s nothing to put in them.